source/docs/flow_control.md

```diff
@@ -280,8 +283,8 @@ The high watermark path is as follows:
    `Network::ConnectionCallbacks::onAboveWriteBufferHighWatermark()`.
  * When `Envoy::Http::CodecClient` receives `onAboveWriteBufferHighWatermark()` it
    calls `onUnderlyingConnectionAboveWriteBufferHighWatermark()` on `codec_`.
- * When `Envoy::Http::ConnectionManagerImpl` receives `onAboveWriteBufferHighWatermark()` it calls
-   `runHighWatermarkCallbacks()` for each stream of the connection.
+ * When `Envoy::Http::Http2::ConnectionImpl` or `Envoy::Http::Http1::ConnectionImpl` receives `onAboveWriteBufferHighWatermark()` it calls
```

Comment:
one other detail is that  `Envoy::Http::Http2::ConnectionImpl` and `Envoy::Http::Http1::ConnectionImpl` receive

`onUnderlyingConnectionAboveWriteBufferHighWatermark` and `onUnderlyingConnectionBelowWriteBufferLowWatermark` not `onAboveWriteBufferHighWatermark`

which might have been part of the typo